src/index.js

```diff
@@ -1,54 +1,66 @@
 /* eslint-disable linebreak-style */
+// eslint-disable-next-line linebreak-style
+/* eslint-disable no-undef */
+/* eslint-disable linebreak-style */
 /* eslint-disable no-unused-vars */
 /* eslint-disable linebreak-style */
 /* "import/no-unresolved": "off" */
 
 // import _ from 'lodash';
```

Comment:
- [ ] Kindly avoid disabling linters to make your code follow standards and best practise.